zebrad/src/components/mempool/downloads.rs

```diff
@@ -373,6 +373,7 @@ where
                 .oneshot(tx::Request::Mempool {
                     transaction: tx.clone(),
                     height: next_height,
+                    skip_checks: None
```

Comment:
Why does guarding this by `#[cfg(any(test, feature = "proptest-impl"))]` not work, and why does it work as it is since `skip_checks` is guarded with the same attribute in `zebra-consensus`?

Reply:
Turns out it doesn't. I have to understand better how feature assignment behaves during compilation. And it's probably not a good idea to do it this way anyway.

Reply:
I used this hack to solve this: https://github.com/zcashfoundation/zebra/blob/9597792567982ae19643eba0d2537d3a81fe5956/zebrad/Cargo.toml#L301-L304.

Reply:
I'm trying to think of something better in general.